tests/rptest/tests/topic_recovery_test.py

```diff
@@ -1154,7 +1154,7 @@ def verify():
 
         wait_until(verify,
                    timeout_sec=timeout.total_seconds(),
-                   backoff_sec=1,
+                   backoff_sec=5,
```

Comment:
i think that if the backoff_sec is affecting the test then there is a deeper problem somewhere else, or a subtle timing issue between the timeout and time it takes to evaluate the predicate.

please describe what the problem is and how this change fixes the problem.

Reply:
We have upload deadline in redpanda set to 10 sec by default.
Here we were waiting just for 6 seconds (because backoff was 1 sec, so we were trying every second. If in 6 attempts nothing changes we just consider it as stable)
So nothing could change in 6 seconds because we have 10 sec upload deadline for not closed logs

Reply:
I will add this to commit message